### DISCUSSION TITLE
Use conventional commits

### DISCUSSION THREAD

**Discussion by helpermethod** [quarkusio/quarkus#29854] [Community]
When reading release notes, it's hard to spot bug fixes amongst new features, dependency updates etc..

While [JReleaser](https://jreleaser.org/guide/latest/configuration/release/github.html#_release_notes) allows you to categorize changelog entries, Quarkus' commits are just not uniform enough to categorize them. Thus I suggest to use [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) to allow automatic categorization by [JReleaser](https://jreleaser.org/guide/latest/configuration/release/changelog.html#_conventional_commits).

/CC @aalmiray

**Comment by aalmiray** [comment]
FWIW I think Quarkus does not use JReleaser for generating its changelog (yet 😉 ). However, following a commit message convention (which ever it may be) it's still a Good Idea. If CC is chosen then there are GitHub apps that can enforce abiding to the rules when PRs are checked.
